ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,282 @@
+            exit 1
+        else
+            logger "${1^} started"
+        fi
+    elif command -v service > /dev/null 2>&1; then
+        if command -v chkconfig > /dev/null 2>&1; then
+            chkconfig ${service_name} on
+        elif command -v update-rc.d > /dev/null 2>&1; then
+            update-rc.d ${service_name} defaults
+        fi
+        service ${service_name} start
+        if [ "$?" != 0 ]; then
+            logger "${1^} could not be started."
+            exit 1
+        else
+            logger "${1^} started"
+        fi
+    else
+        logger "Error: ${1^} could not start. No service manager found on the system."
+        exit 1
+    fi
+}
+
+## Show script usage
+getHelp() {
+
+   echo ""
+   echo "Usage: $0 arguments"
+   echo -e "\t-d   | --debug Shows the complete installation output"
+   echo -e "\t-i   | --ignore-health-check Ignores the health-check"
+   echo -e "\t-h   | --help Shows help"
+   exit $1 # Exit script after printing help
+}
+
+
+## Install the required packages for the installation
+installPrerequisites() {
+    logger "Installing all necessary utilities for the installation..."
+    $sys_type install curl unzip wget libcap -y -q
+    echo -e '[AdoptOpenJDK] \nname=AdoptOpenJDK \nbaseurl=http://adoptopenjdk.jfrog.io/adoptopenjdk/rpm/centos/$releasever/$basearch\nenabled=1\ngpgcheck=1\ngpgkey=https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public' | tee /etc/yum.repos.d/adoptopenjdk.repo
+    $sys_type install adoptopenjdk-11-hotspot -y -q
+    export JAVA_HOME=/usr/
+
+    if [ "$?" != 0 ]; then
+        logger "Error: Prerequisites could not be installed"
+        exit 1
+    else
+        logger "Done"
+    fi
+}
+
+## Add the Wazuh repository
+addWazuhrepo() {
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    logger "Adding the Wazuh repository..."
+    rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH
+    if [ "${PACKAGES_REPOSITORY}" = "prod" ]; then
+        logger "Adding production repository..."
+        echo -e "[wazuh_repo]\ngpgcheck=1\ngpgkey=https://packages.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages.wazuh.com/${WAZUH_MAJOR}.x/yum/\nprotect=1" | tee /etc/yum.repos.d/wazuh.repo
+    elif [ "${PACKAGES_REPOSITORY}" = "dev" ]; then
+        logger "Adding development repository..."
+        echo -e '[wazuh_pre_release]\ngpgcheck=1\ngpgkey=https://packages-dev.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages-dev.wazuh.com/pre-release/yum/\nprotect=1' | tee /etc/yum.repos.d/wazuh.repo
+    fi
+    if [ "$?" != 0 ]; then
+        logger "Error: Wazuh repository could not be added"
+        exit 1
+    else
+        logger "Done"
+    fi
+}
+
+addElasticRepo(){
+    logger "Adding the Elastic repository..."
+    curl https://d3g5vo6xdbdb9a.cloudfront.net/yum/opendistroforelasticsearch-artifacts.repo -o /etc/yum.repos.d/opendistroforelasticsearch-artifacts.repo
```

Comment:
Won't we use our own repo?

Reply:
Not yet, this is planned for the next release.